windows-lb/dbtier.tf

```diff
@@ -0,0 +1,57 @@
+## Copyright © 2021, Oracle and/or its affiliates. 
+## All rights reserved. The Universal Permissive License (UPL), Version 1.0 as shown at http://oss.oracle.com/licenses/upl
+
+resource "oci_core_instance" "MSSQL_Primary" {
```

Comment:
Any reason to prefer MySQL installation over compute as opposed to the MySQL service ?

Reply:
@jeevanjoseph it is not MySQL. It is MS SQL. :)